### PR TITLE
Updated dump/cleanup

### PR DESCRIPTION
**Pull request by GeBo1**
Some newly dumped string in Localizations (already covered, but now synced with latest dump).
Using `planname` assets (needs TextResoureRedirector update)
Fix the remaining `■`  issue.

### CODE REVIEW

File: Translation/en/Text/Formatting.txt [hunk 1]
```diff
@@ -15,3 +15,8 @@ sr:"^■\s*(.*)"=■ $1
 sr:"^「([^」]+)」が達成されました$"="$1" Achieved
 
 r:"^残り([０-９0-9]+)Ｐ$"=$1P Remaining
+
+// When the service is selected:
+#set level 15
+sr:^～(.*)～$=~$1~
```
Comment (SpockBauru) [inline Translation/en/Text/Formatting.txt:21]: If TextResourceRedirector will add those lines with ～text～ and also strip them, is this regex needed?

File: Translation/en/RedirectedResources/assets/abdata/spr/list/50/planname/translation.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+//
+
+// TextResourceRedirector will add these as standard translations with the ～ stripped as well
+
+～ナイトラブプラン～=Night Pool Service
+～アダルトプラン～=BDSM Service
+～ドキドキ密着プラン～=Crowded Train Service
+～不思議体験プラン～=Tentacle Service
+～夜のお散歩プラン～=Night Park Service
+～フォーエバープラン～=Wedding Service
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/spr/list/50/planname/translation.txt:12]: Installed the new HS2_TextResourceRedirector.v1.4.3.1 and seems to not be working, I've sent more details in the Discord.

File: Translation/en/Text/Formatting.txt (hunk 1 above)
Comment (GeBo1) [reply Translation/en/Text/Formatting.txt:21]: I'd like to keep it for future proofing. If/when they add something new it won't pollute the cache this way.

File: Translation/en/RedirectedResources/assets/abdata/spr/list/50/planname/translation.txt (hunk 2 above)
Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/spr/list/50/planname/translation.txt:12]: Sorry, new version posted on discord. I'd adjusted the check to see if it was running under DX or not, but it wasn't woking right.

**Pull request merged**